test/core/client_channel/lb_policy/lb_policy_test_lib.h

```diff
@@ -702,6 +709,7 @@ class LoadBalancingPolicyTest : public ::testing::Test {
       absl::Span<const absl::string_view> addresses,
       const std::map<UniqueTypeName, std::string> call_attributes = {},
       size_t num_iterations = 3, SourceLocation location = SourceLocation()) {
+    ASSERT_NE(picker, nullptr);
```

Comment:
I don't think we need this check, since `GetCompletePicks()` handles this case for us.

Reply:
Gone.